### DISCUSSION TITLE
Install Oversseerr inside the Tautulli LXC : is it possible ? how ?

### DISCUSSION THREAD

**Discussion by MilesTEG1** [tteck/Proxmox#3216] [Help Needed]
Hello,
I'm using for some weeks now, a Plex LXC container (privileged for SMB mount), and a Tautulli LXC.

I'm looking for a way to install Overseerr inside the actual Tautulli LXC to have the two in the same LXC, I'm not wanting to add another LXC to maintain. 

Is it possible? If yes how?

For now, I'm migrating to VM with docker.

Thanks for your scripts, and for any help :)

**Comment by Mellowlynx** [comment]
It might be possible if you install it yourself.

**Reply by MilesTEG1** [reply]
> It might be possible if you install it yourself.

How can I do that?

**Reply by Mellowlynx** [reply]
Read here: https://docs.overseerr.dev/getting-started/installation#linux
Note that this not supported situation, so all at your own risk.
